### PR TITLE
hwloc: update to 2.1.0

### PR DESCRIPTION
**Pull request by flyn-org**
Signed-off-by: W. Michael Petullo <mike@flyn.org>

Maintainer: me
Compile tested: x86_64 master bba6646b
Description:

### CODE REVIEW

File: libs/hwloc/Makefile [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=https://download.open-mpi.org/release/$(PKG_NAME)/v2.0
-PKG_HASH:=e393aaf39e576b329a2bff3096d9618d4e39f416874390b58e6573349554c725
+PKG_SOURCE_URL:=https://download.open-mpi.org/release/$(PKG_NAME)/v2.1
+PKG_HASH:=19429752f772cf68321196970ffb10dafd7e02ab38d2b3382b157c78efd10862
 
 PKG_LICENSE:=BSD-3-Clause
```
Comment (neheb) [inline libs/hwloc/Makefile]: PKG_LICENSE_FILES:=COPYING

File: libs/hwloc/Makefile [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
 PKG_MAINTAINER:=W. Michael Petullo <mike@flyn.org>
```
Comment (neheb) [inline libs/hwloc/Makefile]: Place above PKG_LICENSE

**Comment by neheb** [comment]
PKG_FIXUP:=autoreconf does not seem necessary.

**Comment by flyn-org** [comment]
Added PKG_LICENSE_FILES and removed PKG_FIXUP.

**Pull request merged**